core/engine/src/value/conversions/nullable.rs

```diff
@@ -0,0 +1,283 @@
+//! A value that can be `null` in JavaScript, but not `undefined`.
+
+use crate::value::{TryFromJs, TryIntoJs};
+use boa_engine::{Context, JsResult, JsValue};
+
+mod tests;
```

Comment:
```suggestion
#[cfg(test)]
mod tests;
```

I see that the `cfg` is in the test module... which is the first time I'm seeing that, didn't know you could do that :laughing: , but I think we should be consistent with other way we declare tests modules.

(assumption): It might be slightly faster for annotation to be outside, since we don't have to check in the module to eliminate the module from build.